activetable/templates/html/activetable.html

```diff
@@ -0,0 +1,47 @@
+<div class="activetable_block">
+  {% if help_text %}
+  <div class="activetable-help" style="width: {{ total_width }}px;">
+    <button id="activetable-help-button">+help</button>
+    <p id="activetable-help-text">{{ help_text }}</p>
+  </div>
+  {% endif %}
+  {% if thead %}
+  <table id="activetable">
+    <colgroup>
+      {% for width in column_widths %}<col style="width: {{ width }}px;">{% endfor %}
+    </colgroup>
+    <thead>
+      <tr style="height: {{ head_height }}px;">
+        {% for cell in thead %}<th>{{ cell }}</th>{% endfor %}
+      </tr>
+    </thead>
+    <tbody>
+      {% for row in tbody %}
+      <tr class="{{ row.class }}" style="height: {{ row.height }}px;">
+        {% for cell in row.cells %}
+        <td class="{{ cell.classes }}" id="{{ cell.id }}">
+          {% if cell.is_static %}
+          {{ cell.value }}
+          {% else %}
+          <input type="text" style="height: {{ cell.height }}px;" size=1
```

Comment:
@cptvitamin Fine, if a `label` tag is better, I'll go with that.  It works fine with the `sr` class.

I hesitated a bit since the only way to link the label and the input is using an id for the input, so I'm aggravating the problem with multiple ActiveTables on one page not being supported.  If we want to support multiple tables, we'll have to add some hash to all ids anyway, so this really won't be an additional issue.  (I can't include the input in the label because of the `sr` class.)